firebase/firestore/invite_admin.ts

```diff
@@ -0,0 +1,31 @@
+import firebaseApp from "../firebaseApp";
+import "firebase/firestore";
+import { AdminInvite } from "../../types/schema";
+
+const db = firebaseApp.firestore();
+const adminInvitesCollection = db.collection("admin_invites");
+
+/**
+ * Adds the given admin data to firestore
+ */
+export const addAdminInvite = async (admin: AdminInvite) => {
```

Comment:
Could you make sure you don't add duplicate admin invites if an invite with the given email already exists